### ISSUE TITLE
Error using org.dspace.authenticate.CASAuthentication

### ISSUE THREAD

**Issue by CrisGuzmanS** [DSpace/DSpace#8703]
Hello @tdonohue . I would like to know if is CAS Authentication is avaible. i already did these steps: https://wiki.lyrasis.org/display/DSDOC6x/Authentication+Plugins but i got an error. `("message": "Cannot load plugin class: java.lang.ClassNotFoundException: org.dspace.authenticate.CASAuthentication")`

1.  Install DSpace 7.5
2. Configure configuration according to this documentation: https://wiki.lyrasis.org/display/DSDOC6x/Authentication+Plugins
3. go to http://localhost:8080/server/
4. you will get the error described above:
![image](https://user-images.githubusercontent.com/51240376/223579346-0d39ec5f-db6f-4c9e-9da5-c681782f2627.png)

**Expected behavior**
Avoid error when setting up the configuration.

**Comment by tdonohue** [comment]
@CrisGuzmanS : No, DSpace doesn't support CASAuthentication.  It's not listed in our official docs as an authentication plugin: https://wiki.lyrasis.org/display/DSDOC7x/Authentication+Plugins

The docs you linked to are on a wiki page created by a community member who must have customized their DSpace to support CAS.  But, that work never came back into the main codebase.

There were some (very old) discussions about adding a CAS plugin in this old ticket: https://github.com/DSpace/DSpace/issues/4400  However, that ticket was closed because of inactivity.  It was never adopted into DSpace.  There was an old PR related to this here https://github.com/DSpace/DSpace/pull/804 (but again it was closed because of inactivity).

Overall, the summary is that DSpace has never supported CAS authentication.  If someone wanted to contribute a plugin for CAS authentication support, we could consider it.

**Comment by erseco** [comment]
Hi @tdonohue, I see that CAS authentication has never been officially supported in DSpace, and previous discussions and PRs were closed due to inactivity. I’d like to ask if there are any current plans or discussions about supporting CAS authentication in future versions beyond DSpace 9. Has there been any recent interest from the community or contributors in implementing this feature? Thanks!

**Comment by tdonohue** [comment]
@erseco : At this time, I have not heard of anyone in the community interested in implementing this feature.  All authentication plugins for DSpace have been provided by community members interested in supporting that authentication option.  So, if someone were to build a CAS authentication plugin, then we would review/test it and try to add it to the system.  The issue is simply that no one has shown an interest in such a plugin.

As an example, we'll be adding SAML2 support as of DSpace 9.0 because a community member decided to build a SAML2 plugin (see #9470).  The only way CAS will ever be supported is if a community member decides to build a plugin.  We don't have a centralized development team for DSpace -- all new features are donated by community members.

**Comment by erseco** [comment]
Hi @tdonohue,

Thanks a lot for your response! SAML 2.0 works perfectly for my needs, so that’s great news. Appreciate the clarification!
